tests/test_code.py

```diff
@@ -55,3 +56,27 @@ def test_RegionCode_hierarchy_attribute():
     )
 
     assert reg.hierarchy == "R5"
+
+
+def test_RegionCode_iso3_code():
+    reg = RegionCode(
+        name="OECD",
```

Comment:
I suggest to use a more "correct" test, e.g. Western Europe that has France, Germany, Spain, Portugal...